data/themes/darktable-elegant-dark.css

```diff
@@ -37,7 +37,7 @@
 /* Modules box (plugins) */
 @define-color plugin_bg_color @grey_35;
 @define-color plugin_fg_color @grey_85;
-@define-color section_label @grey_75;
+@define-color section_label @fg_color;
```

Comment:
I agree with the idea. Those section labels needs indeed to be brighter. But I'm not have same section labels color like module ones. Like in typical writing document, subtitles are not same as head parent titles. I suggest to simply apply some percent to have something near and consistent. I suggest that change instead:
`
@define-color section_label shade(@plugin_fg_color, 0.9);`

This line need to be applied only in darktable.css file and so you can remove section_label line in grey and dark one, as it will inherit correctly (tested.